db/src/models/ticket_types.rs

```diff
@@ -584,14 +584,16 @@ impl TicketType {
         )
     }
 
-    pub fn is_event_not_draft(
+    pub fn is_event_available_for_sale(
         ticket_type_id: &Uuid,
         conn: &PgConnection,
     ) -> Result<bool, DatabaseError> {
         let valid_ticket_count: i64 = ticket_types::table
             .inner_join(events::table)
             .filter(ticket_types::id.eq(ticket_type_id))
             .filter(events::status.ne(EventStatus::Draft.to_string()))
+            .filter(events::status.ne(EventStatus::Draft.to_string()))
```

Comment:
Is this a duplicate?
```suggestion
            .filter(events::cancelled_at.is_null())
```

Reply:
Good catch. Must have copied/pasted and then not checked